### ISSUE TITLE
Accessibility of the components

### ISSUE THREAD

**Issue by felixzapata** [angular-ui/bootstrap#1438]
hi, would you accept some pull requests related with the accessibility of the components? For example, the modal lacks of keyboard support.

Thanks.
